docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Do we really need to mention the <code>activations</code> attribute?</p><p>the whole point is that <code>AlertActivations </code> basically wraps a list of <code>AlertActivation</code> objects, so it should be "invisible" to the user.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
@TomerShor we need object to wrap all alerts to do `group_by` and `aggreagte_by`. Also, user still can iterate over it. What are you suggesting then?

Reply:
I suggested that you don't mention that the `AlertActivations` object has an underlying `activations` attribute, as it is not actually used by the client.
You can still iterate or get a single activation from the list without accessing this attribute

Reply:
@TomerShor why not being clear with user?